src/App.jsx

```diff
@@ -1,13 +1,88 @@
 import { useEffect, useState } from 'react'
-
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import './App.css'
-import TodoList from './TodoList'
-import AddTodoForm from '../AddTodoForm'
+import TodoList from './components/TodoList'
+import AddTodoForm from './components/AddTodoForm'
+
+function AscendingSort(objectA, objectB) {
+  if (objectA < objectB) {
+    return -1
+  } else if (objectA > objectB) {
+    return 1
+  } else {
+    return 0
+  }
+}
+
+function DescendingSort(objectA, objectB) {
+  if (objectA > objectB) {
+    return -1
+  } else if (objectA < objectB) {
+    return 1
+  } else {
+    return 0
+  }
+}
 
 function App() {
-  //  const useSemiPersistentState = () => {
   let [todoList, setTodoList] = useState([])
   let [isLoading, setIsLoading] = useState(true)
+  let [sortAscend, setSortAscend] = useState(true)
+
+  async function fetchData() {
+    const options = {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`
+      }
+    }
+    const url = `https://api.airtable.com/v0/${
+      import.meta.env.VITE_AIRTABLE_BASE_ID
+    }/${
+      import.meta.env.VITE_TABLE_NAME
+    }?view=Grid%20view&sort[0][field]=Title&sort[0][direction]=asc`
+
+    try {
+      const response = await fetch(url, options)
+
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`)
+      }
+
+      const data = await response.json()
+
+      const todos = data.records.map((record) => ({
+        id: record.id,
+        title: record.fields.Title,
+        createdTime: record.createdTime
+      }))
+
+      setTodoList(todos)
+      setIsLoading(false)
+    } catch (error) {}
+  }
+
+  useEffect(() => {
+    fetchData()
+  }, [])
+  function sortTodos(todos, sortAscend) {
+    return todos.sort((objectA, objectB) => {
+      // if (sortAscend) {
```

Comment:
Please make sure to remove commented code to keep active code clear and readable – let me know if you have any questions about this!